model_analyzer/config/generate/base_model_config_generator.py

```diff
@@ -162,19 +162,6 @@ def make_model_config(
         """
         Loads the base model config from the model repository, and then applies the
         parameters in the param_combo on top to create and return a new model config
-
```

Comment:
Do we still need parameters now that we have type checking? They were incorrect, which is why I removed them

Reply:
I see they were incorrect. It would be nice if they were correct. The type checking tells us what type is passed in, but doesn't give us all the information. For example, if I pass in a bool, stating it is a bool doesn't tell me what that bool means.